rpm/telepathy-qt5.spec

```diff
@@ -50,7 +50,6 @@ Requires:   %{name} = %{version}-%{release}
 Requires:   telepathy-qt5 = %{version}
```

Comment:
This line looks strange. The line above should be enough. Could this be removed too?

Reply:
Strange indeed. Removing.